server/graphql/common/user.ts

```diff
@@ -81,12 +83,11 @@ export const hasSeenLatestChangelogEntry = async (user: typeof models.User): Pro
     });
 
     latestChangelogUpdatePublishDate = latestChangelogUpdate?.publishedAt;
+    if (latestChangelogUpdatePublishDate) {
+      return true;
+    }
```

Comment:
I don't get this condition, why not check against the `user?.changelogViewDate` here?

Reply:
@Betree : If this condition is not preset the changelog notification will appear without any new changelogs present. This will happen when we enable the changelog feature flag, but before posting the first update. 😄 I know it's a kinda condition which only helps initially when there's no changelog updates in the db, but I want to handle that condition so users don't get unnecessary notifications.

Reply:
Shouldn't that be:

```suggestion
    if (!latestChangelogUpdatePublishDate) {
      return true;
    }
```

Reply:
@Betree : Ops, yes you are right. 🤯